resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -88,33 +88,27 @@
                             <th scope="col">Start Time</th>
                             <th scope="col">Lesson Type</th>
                             <th scope="col">Instructor/Mentor</th>
+                            <th scope="col">View/Modify</th>
                         </tr>
                     </thead>
                     <tbody>
-                        @if($setmore_data_stale)
-                            <tr class="table-warning">
-                                <td colspan="4">
-                                    <i class="fas fa-exclamation-triangle"></i> Our connection with Setmore is malfunctioning - reservation data may be inaccurate <i class="fas fa-exclamation-triangle"></i>
-                                </td>
-                            </tr>
-                        @endif
-                        @if(count($setmore_appointments) > 0)
-                            @foreach($setmore_appointments as $sm_res)
+                        @if(count($ea_appointments) > 0)
+                            @foreach($ea_appointments as $ea_appointment)
                                 <tr>
-                                    <td scope="col">{{ $sm_res->res_date }}</td>
-                                    <td scope="col">{{ $sm_res->res_time }} ET</td>
-                                    <td scope="col">{{ $sm_res->service_description }}</td>
-                                    <td scope="col">{{ $sm_res->staff_name }}</td>
+                                    <td scope="col">{{ $ea_appointment->res_date }}</td>
+                                    <td scope="col">{{ $ea_appointment->res_time }} ET</td>
+                                    <td scope="col">{{ $ea_appointment->service_description }}</td>
+                                    <td scope="col">{{ $ea_appointment->staff_name }}</td>
+                                    <td scope="col"><a href="https://scheduling.ztlartcc.org/index.php/booking/reschedule/{{ $ea_appointment->link_token }}" target="_blank" class="btn btn-primary simple-tooltip" data-toggle="tooltip" title="View"><i class="fas fa-edit fa-fw"></i></a></td>
```

Comment:
Could we get the autofill on this too? Not essential but it would be nice, and it should already be there from the existing script.

Reply:
Done

Reply:
Looks great!